### DISCUSSION TITLE
Vercel & Prisma - Unable to connect via SSL

### DISCUSSION THREAD

**Discussion by ytsruh** [vercel/vercel#8951] [General]
Hoping for some advice, not sure if this is more of a Prisma issue or a Vercel issue. 

I'm able to read an SSL certificate and connect to my database locally. However when I use the same relative path in a deployed envinroment to Vercel, the file can no longer be read & there a connection is refused.

My assumption is that the relative path to the prisma folder is incorrect after Vercel has done its build step. However I've no idea how to locate where the certiticate would be stored.

Does anybody have any advice or guidance?

**Comment by amyegan** [comment]
Hi @ytsruh! I know you can't share a fully working example, but can you share something close to a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) with the private security info redacted? That will help give me a better understanding of how you're trying to connect. If this is based on a template, article, or tutorial, then a link to that resource would be helpful as well 🙏

**Reply by ytsruh** [reply]
Super sorry I didnt reply to this. I went on vacation & then.....you know....Christmas.

I absoluetly will pull together a reproducable example with redactions / fake security creds. Give me a few days
